administration/src/util/result.ts

```diff
@@ -0,0 +1,96 @@
+type ResultError = { type: string }
```

Comment:
I think it would be nice to have the possible types as string literal types here

Reply:
This is the very very very generic case. This is just the type-shape of Errors in general. In another file an error type specific for the administration frontend could be defined.